arangod/IResearch/IResearchInvertedIndexMeta.cpp

```diff
@@ -126,12 +126,11 @@ const IResearchInvertedIndexMeta& IResearchInvertedIndexMeta::DEFAULT() {
   return meta;
 }
 
-IResearchInvertedIndexMeta::IResearchInvertedIndexMeta()
-    : _indexingContext{
-          std::make_unique<IResearchInvertedIndexMetaIndexingContext>(this,
-                                                                      false)} {
+IResearchInvertedIndexMeta::IResearchInvertedIndexMeta() {
   _analyzers[0] = FieldMeta::identity();
   _primitiveOffset = _analyzers.size();
+  _indexingContext =
```

Comment:
Y?

Reply:
We pass this to ctor of indexing context.
I faced with bug, when tried to use this in indexing context ctor